### DISCUSSION TITLE
remark-definition-list - how to set defListHastHandlers

### DISCUSSION THREAD

**Discussion by bartosz-the-coder** [remarkjs/remark#888] [Q&A] [answered]
Documentation of `remark-definition-list` shows that `defListHastHandlers` object has to passed as handlers of  `remark-rehype`.

Unfortunately I'm not able to find a way to do so.
Without mentioned handlers, a definition list renders as divs and a paragraph.

``` js
const html = unified()
  .use(remarkParse)
  .use(remarkDefinitionList)
  .use(remarkRehype, {
    **handlers: Object.assign({}, defListHastHandlers),**
  })
  .use(rehypeStringify)
  .process(md);
```

I would like to ask for any help to make a definition list work.

Thanks!

**Comment by wooorm** [comment, marked as answer]
`remark-definition-list` is not maintained by us here. You’re probably better off asking the people that maintain it (because I have no clue).
